src/app/features/home/home.component.html

```diff
@@ -1,10 +1,567 @@
+            class="px-16 py-4 rounded-2xl bg-[white] text-blue-950 mt-4 transition hover:bg-white/75"
+          >
+            {{ "translate_about_platform" | translate }}
+          </button>
+        </div>
+        <ul
+          class="mt-24 pb-32 md:pb-16 md:ps-8 text-white flex flex-col md:flex-row gap-5"
+        >
+          <li class="flex items-center gap-3">
+            <i class="fa-sharp fa-solid fa-sparkles fa-2xl"></i>
+            <p class="text-2xl">
+              {{ "translate_new_concept" | translate }}
+            </p>
+          </li>
+          <li class="flex items-center gap-3">
+            <i class="fa-solid fa-briefcase fa-2xl"></i>
+            <p class="text-2xl">
+              {{ "translate_freedom_of_action" | translate }}
+            </p>
+          </li>
+          <li class="flex items-center gap-3">
+            <i class="fa-solid fa-clock fa-2xl"></i>
+            <p class="text-2xl">
+              {{ "translate_appreciation_of_time" | translate }}
+            </p>
+          </li>
+          <li class="flex items-center gap-3">
+            <i class="fa-solid fa-hands-holding-circle fa-2xl"></i>
+            <p class="text-2xl">
+              {{ "translate_inclusive_services" | translate }}
+            </p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <!-- Request-Price section -->
+    <section>
+      <div class="container pt-5 relative">
+        <h1
+          class="text-4xl font-bold text-center my-10 before:absolute before:block before:w-14 before:h-1 before:bg-teal-500 before:rounded-sm before:bottom-[104rem] md:before:bottom-[53.5rem] before:left-[45%] md:before:left-[50%]"
+        >
+          {{ "translate_why_order_a_better_price" | translate }}
+        </h1>
+        <div class="mt-5 grid grid-cols-1 md:grid-cols-2 gap-5">
+          <div class="relative">
+            <p class="text-3xl text-teal-500">
+              {{ "translate_info" | translate }}
+            </p>
+            <h1
+              class="mt-4 text-4xl font-bold before:absolute before:block before:w-14 before:h-1 before:bg-teal-500 before:rounded-sm before:bottom-[39.7rem] md:before:bottom-[43.7rem] after:absolute after:block after:w-14 after:h-1 after:bg-teal-500 after:rounded-sm after:bottom-[39rem] md:after:bottom-[43rem] after:-translate-x-4 ltr:after:translate-x-4"
+            >
+              {{ "translate_why_order_a_price" | translate }}
+            </h1>
+            <div class="mt-20 flex flex-col gap-2 text-slate-400 text-xl">
+              <p>{{ "translate_why_order_a_price_first" | translate }}</p>
+              <p>{{ "translate_why_order_a_price_second" | translate }}</p>
+              <p>{{ "translate_why_order_a_price_third" | translate }}</p>
+              <p>{{ "translate_why_order_a_price_fourth" | translate }}</p>
+            </div>
+            <div class="relative w-full mt-20">
+              <img
+                src="../../../assets/images/team-work.jpg"
```

Comment:
```suggestion
                src="assets/images/team-work.jpg"
```
Do the same for similar.